lua_modules/mcp23017/mcp23017.lua

```diff
@@ -17,24 +17,15 @@
 	@version 1.0.0
 ]]
 
-local i2c, string, issetBit, setBit, clearBit, error =
-      i2c, string, bit.isset, bit.set, bit.clear, error
-local isINPUT, isGPB, isHIGH = true, true, true
+local i2c, string, issetBit, setBit, clearBit =
+      i2c, string, bit.isset, bit.set, bit.clear
 
 -- registers (not used registers are commented out)
-local MCP23017_IODIRA = 0x00
-local MCP23017_IODIRB = 0x01
-local MCP23017_DEFVALA = 0x06
-local MCP23017_DEFVALB = 0x07
-local MCP23017_GPIOA = 0x12
-local MCP23017_GPIOB = 0x13
 --[[
```

Comment:
@galjonsfigur If someone uses the MCP23017, they will have the datasheet too. Theoretically the registers can be removed. I wouldn't insert a link because the link is in short time invalid :thinking: